server/src/main.rs

```diff
@@ -25,8 +27,9 @@ async fn game_count(model: web::Data<Model>) -> impl Responder {
 
 /// POST /game/create This
 /// function will be called from a post request
-///
-async fn post_game(form: web::Form<FormData>) -> impl Responder {
+async fn post_game(form: web::Form<FormData>, model: web::Data<Mutex<Model>>) -> impl Responder {
+    let mut g_model = model.lock().unwrap();
+    g_model.add_game(game::init(&form.game_name));
```

Comment:
What is `g_` in `g_model`?

Could `model` be renamed like `mutex` or `data` and then `let mut model = mutex.loc().unwrap()` or something? I dont know what names make sense, I am just saying you can free up `model` if you name the prior thing something else.

Reply:
I renamed it to `data` will make a pull request now

Reply:
I also renamed the app_data from `g_model` to global state